package.json

```diff
@@ -37,26 +38,40 @@
     "url": "https://github.com/iTwin/insights-client"
   },
   "dependencies": {
-    "cross-fetch": "^3.1.4"
+    "cross-fetch": "^3.1.4",
+    "dom-parser": "^0.1.6",
```

Comment:
Is the `dom-parser` still needed if you are using `fast-xml-parser`? If not - `npm uninstall dom-parser`.

Reply:
uninstalled